### PR TITLE
Task 27 forrgot

### PR DESCRIPTION
**Pull request by OlegPanasyuk**

### CODE REVIEW

File: src/components/main/index.js [hunk 1]
```diff
@@ -6,15 +6,21 @@ import { GettingStarted } from '../help/gettingStarted';
 import { Profile } from '../../containers/profile';
 import { Trobleshooting } from '../help/troubleshooting';
 import { CourseTaking } from '../help/courseTaking'
+import ResetPassword from '../../containers/auth/resetPassword';
 
-export const Main = () => {
+export const Main = ({match}) => {
+  console.log(match);
```
Comment (slavasgod) [inline src/components/main/index.js]: Remove clg

File: src/containers/auth/forgotPassword/index.jsx [hunk 2]
```diff
@@ -0,0 +1,117 @@
...
+    status: null,
+    data: null,
+    email: null
+  }
+
+  refInput = React.createRef();
+
+  send = async (email) => {
+    const response = await forgotPasswordRequest({ email });
+    console.log(response);
```
Comment (slavasgod) [inline src/containers/auth/forgotPassword/index.jsx]: Remove clg

File: src/containers/auth/forgotPassword/index.jsx [hunk 3]
```diff
@@ -0,0 +1,117 @@
...
+              id="forgotPassWordEmail"
+              className={input}
+              ref={this.refInput}
+              placeholder='Enter email'
+              onChange={this.onChangeInput}
+            />
+          </div>)
+          }
+          {
+            (status < 300) && (
```
Comment (slavasgod) [inline src/containers/auth/forgotPassword/index.jsx]: Use ternary expressions

File: src/containers/auth/forgotPassword/styles.module.scss [hunk 4]
```diff
@@ -0,0 +1,75 @@
...
+        padding: 15px;
+        box-sizing: border-box;
+    }
+
+    .row {
+        display: flex;
+        margin: 15px 0 0 0;
+    }
+
+    .rowTop {
```
Comment (slavasgod) [inline src/containers/auth/forgotPassword/styles.module.scss:34]: In css use codestyle like "row_top"

File: src/services/auth.js [hunk 5]
```diff
@@ -35,3 +35,48 @@ export const googleLoginRequest = async data => {
...
+
+export const forgotPasswordRequest = async data => {
+  try {
+    const response = await axios.post('api/users/reset', data);
+    return response;
+  } catch (e) {
+    return {
+      status: 400,
+      data: 'Bad email',
+    };
```
Comment (slavasgod) [inline src/services/auth.js:47]: Replace this on validation on backend side

File: src/containers/auth/forgotPassword/index.jsx (hunk 3 above)
Comment (OlegPanasyuk) [reply src/containers/auth/forgotPassword/index.jsx]: Ternary operators go to hell

File: src/services/auth.js (hunk 5 above)
Comment (OlegPanasyuk) [reply src/services/auth.js:47]: This is a problem with axios with Try Catch. It throws Error Exception when server sends error message

**Pull request merged**